src/registryDAO/types.mligo

```diff
@@ -52,3 +52,14 @@ type lookup_registry_param =
   }
 
 type lookup_registry_view = (registry_key * (registry_value option)) contract
+
+type initial_registryDAO_storage =
+  { trivial_data : initial_data
```

Comment:
`trivial_data` here is not the best name, as `registryDAO` and `trivialDAO` are both (different) instances of baseDAO, a more appropriate name could be `base_data` for example